2017/OCT/OCT17/CHEFGP.py

```diff
@@ -0,0 +1,64 @@
+t = int(input())
+while t > 0:
+    s = input()
+    w = ""
+    x, y = map(int, input().split())
+    na = s.count("a")
+    nb = s.count("b")
+    if na > nb:
+        smax = "a"
+        smin = "b"
+    else:
+        smax = "b"
+        smin = "a"
+        tamp = na
+        na = nb
+        nb = tamp
+        tamp = x
+        x = y
+        y = tamp
+    h = ""
+    xa = na/x
+    mini = int(min(xa, nb))
+    h = h + smax * x
+    h = h + smin
+    w = w + h * mini
+    ra = na - mini * x
+    rb = nb - mini
+    if ra < x and ra > 0:
+        w = w + smax * ra
+        if rb != 0:
+            w = w + smin
+            rb -= 1;
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpm5bfynyn/2017/OCT/OCT17/CHEFGP.py
+++ b/tmp/tmpm5bfynyn/2017/OCT/OCT17/CHEFGP.py
@@ -29,7 +29,7 @@
         w = w + smax * ra
         if rb != 0:
             w = w + smin
-            rb -= 1;
+            rb -= 1
     elif ra >= x:
         k = x
         i = 0
```